frontend/src/components/learner/browser/CourseCard.tsx

```diff
@@ -25,6 +42,52 @@ const CourseCard = ({ course }: CourseCardProps): React.ReactElement => {
     ) || 0;
   const expanded = focused || hovered;
 
+  const StatusTag = () => {
```

Comment:
Might be better to pass `status` as a prop, that way this component can be defined out of line.